app/controllers/responsePaginationHelper.js

```diff
@@ -0,0 +1,17 @@
+exports.parseOffset = query => {
+  query.page = Number.parseInt(query.page);
+  query.page = Number.isSafeInteger(query.page) ? query.page : 0;
+  return query.page * exports.parsePageLimit(query);
+};
+
+exports.parsePageLimit = query => {
+  const limit = Number.parseInt(query.limit);
+  if (Number.isInteger(limit)) return limit;
+
+  const environmentLimit = Number.parseInt(process.env.DEFAULT_ITEMS_PER_PAGE);
```

Comment:
When the define DEFAULT_ITEMS_PER_PAGE you can set default value.  Example
`items_per_page : process.env.DEFAULT_ITEMS_PER_PAGE || 8`